src/helm/common/request.py

```diff
@@ -72,6 +72,34 @@ def model_engine(self) -> str:
         return model.engine
 
 
+@dataclass(frozen=True)
+class TextToImageRequest(Request):
```

Comment:
This causes `Request` to not round-trip to JSON - see the comments under `TextToImageAdapterSpec`.